This is handled automatically nowadays, and gthread-2.0 is no longer in LIBS.

--- origsrc/gtkhtml-3.32.2/components/editor/main.c	2010-09-13 07:16:10.000000000 -0500
+++ src/gtkhtml-3.32.2/components/editor/main.c	2013-08-02 14:57:26.881878500 -0500
@@ -373,8 +373,6 @@ main (gint argc, gchar **argv)
 	bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 	textdomain (GETTEXT_PACKAGE);
 
-	g_thread_init (NULL);
-
 	gtk_init (&argc, &argv);
 
 	editor = gtkhtml_editor_new ();
